api/user/delete.py

```
@@ -1,27 +1,32 @@
+#   FUNCTION
+#
+
+# FUNCTION -> ROUTER
+router = APIRouter()
+
+# FUNCTION -> DECLARATION
+@router.post("/api/user/delete")
+async def output(request: Request, response: Response) -> JSONResponse:
+    # DECLARATION -> EXTENSIONS
+    from website.extensions import database
+    await asyncio.gather(
```

Ruff (0.11.9)

`asyncio` may be undefined, or defined from star imports

(F405)